ccm_web/server/src/api/api.admin.handler.ts

```diff
@@ -25,9 +39,13 @@ export class AdminApiHandler {
   requestor: CanvasRequestor
   userLoginId: string
 
-  constructor (requestor: CanvasRequestor, userLoginId: string) {
+  constructor (requestor: CanvasRequestor, userLoginId?: string) {
     this.requestor = requestor
-    this.userLoginId = userLoginId
+    if (typeof userLoginId !== 'undefined') {
```

Comment:
@lsloan, you should be able to just directly check if it is `undefined`. Also, this could be one line:
```
this.userLoginId = userLoginId !== undefined ? `"${userLoginId}"` : '(undefined)'
```

Reply:
🏁